common/src/test/scala/org/apache/celeborn/common/rpc/netty/NettyRpcEnvSuite.scala

```diff
@@ -42,7 +43,7 @@ class NettyRpcEnvSuite extends RpcEnvSuite with TimeLimits {
       port: Int,
       clientMode: Boolean = false): RpcEnv = {
     val config = RpcEnvConfig(
-      conf,
+      conf.set(s"celeborn.$transportModule.io.mode", IOMode.NIO.name()),
```

Comment:
The test case `port conflict` has exception that `io.netty.channel.unix.Errors$NativeIoException: bind(..) failed: Address already in use` for using default `EPOLL` io mode.